framework/src/media/audio/audio_manager.c

```diff
@@ -114,35 +124,35 @@ audio_manager_result_t find_input_audio_card(void)
 	uint8_t card_id = -1;
 	uint16_t device_id = -1;
 	char type;
-	struct dirent *dir_entry;
-	DIR *dir_info;
+	struct dirent* dir_entry;
+	DIR* dir_info;
 
 	if (!(dir_info = opendir("/dev/audio/"))) {
 		return AUDIO_MANAGER_DEVICE_FAIL;
 	}
 
 	while ((dir_entry = readdir(dir_info)) != NULL) {
 		// TODO: Add cases for various drivers. Currently, identify 'pcm' drivers only.
-		if ((dir_entry->d_name[0] != 'p')
-			|| (dir_entry->d_name[1] != 'c')
-			|| (dir_entry->d_name[2] != 'm')
+		if ((dir_entry->d_name[0] != 'p') || (dir_entry->d_name[1] != 'c') || (dir_entry->d_name[2] != 'm')
 			|| (sscanf(&dir_entry->d_name[3], "C%uD%u%c", &card_id, &device_id, &type) != 3)) {
 			continue;
 		}
 
 		if ((type == 'c') && (card_id < CONFIG_AUDIO_MAX_INPUT_CARD_NUM) && (device_id < MAX_AUDIO_DEVICE_NUM)) {
-			snprintf(g_audio_in_cards[card_id].card_path, strlen("/dev/audio/") + strlen(dir_entry->d_name) + 1, "%s%s", "/dev/audio/", dir_entry->d_name);
+			snprintf(g_audio_in_cards[card_id].card_path, strlen("/dev/audio/") + strlen(dir_entry->d_name) + 1,
+					  "%s%s", "/dev/audio/", dir_entry->d_name);
 
 			g_actual_audio_in_card_id = card_id;
 			g_audio_in_cards[card_id].status = AUDIO_CARD_READY;
 			g_audio_in_cards[card_id].device_id = device_id;
 			g_audio_in_card_num++;
-			audvdbg("Found an input audio card, id=%d, count=%d,  card_path=%s\n", card_id, g_audio_in_card_num, g_audio_in_cards[card_id].card_path);
+			audvdbg("Found an input audio card, id=%d, count=%d,  card_path=%s\n", card_id, g_audio_in_card_num,
+					g_audio_in_cards[card_id].card_path);
 		}
 	}
 
 	if (!g_audio_in_card_num) {
-		auddbg("Wrong g_audio_in_card_num\n");
+		audvdbg("Wrong g_audio_in_card_num\n");
```

Comment:
for error case, auddbg is right(of course meddbg is correct, but let chanhee fix it later.)